Advance_1/index.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+    /* Class to call the API. */
```

Comment:
Class comment should be like this and mention in details what this class is doing ->
```
/**
 * Comments ...
 * Comments....
 */
```